recipe/conda_forge_ci_setup/build_utils.py

```diff
@@ -135,17 +136,8 @@ def upload_package(feedstock_root, recipe_root, config_file):
                 print("Uploading to conda-forge with source channel '{}' is not allowed".format(source_channel))
                 return
 
-    upload_script_path = os.path.join(recipe_root, 'conda_forge_ci_setup', 'upload_or_check_non_existence.py')
```

Comment:
Why remove this?

Reply:
See my comments below. Basically, patching this one module is breaking the CI of this recipe itself. We can put an option in smithy to allow people to specify a path to this module to install the tools from or something like that.